.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+Documentation:
+   Enabled: false
```

Comment:
What does happen when you run the command that the card mentions for rubocop? ie: `bundle exec rubocop app spec -R`

Reply:
When I run `invalid option: -R` it responds: "invalid option: -R".

Without -R parameter: 
> Inspecting 7 files
> .......
>7 files inspected, no offenses detected

Reply:
Exactly. The thing is the -R parameter is now deprecated, but we still needthe functionality that were provided by it. The -R modifier enabled the Rails cops of Rubocop. Cops are the rules that Rubocop has for checking issues in your code:

https://github.com/rubocop-hq/rubocop/blob/master/manual/cops.md

So Rails cops were cops that checked for things that were Rails-specific. So now that we don't have the -R parameter, this checks are missing, so it's up to you to configure rubocop to have again this functionality.

Reply:
maybe using this gem: `'rubocop-rails'` ?